editor/src/clj/editor/game_object.clj

```diff
@@ -161,7 +167,7 @@
                             (let [proj-path->resource-property-build-target (bt/make-proj-path->build-target resource-property-build-targets)]
                               (game-object-common/referenced-component-instance-data build-resource ddf-message pose proj-path->resource-property-build-target)))
             build-target (assoc build-target :instance-data instance-data)]
-        [(bt/with-content-hash build-target)]))))
+        [(bt/with-content-hash build-target)])))))
```

Comment:
Style: Indentation is off for this function. **Code > Auto-Indent Lines** should sort it out!